platform-ui-skin/src/main/webapp/skin/less/social/skin/StreamPage/Style.less

```diff
@@ -109,7 +109,7 @@
           box-sizing: border-box;
           max-width: 100%!important;
           width: 100%!important;
-          padding: 0 15px!important;
+          padding: 15px 15px 0 15px!important;
```

Comment:
I think the stream style is already fixed, can you check with develop branch please, you have conflicts ;-)

Reply:
done, thanks a lot !